_wallets/unstoppable.md

```diff
@@ -0,0 +1,42 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+id: unstoppable
+title: "Unstoppable Wallet"
+titleshort: "Unstoppable"
+compat: "mobile android ios"
+level: 2
+platform:
+  - mobile:
+    name: mobile
+    default: &DEFAULT
+      text: "walletunstoppable"
+      link: "https://unstoppable.money"
+      source: "https://github.com/horizontalsystems"
```

Comment:
This should be a direct link to the repository containing the wallet's source code.

Reply:
> This should be a direct link to the repository containing the wallet's source code.

The reason I linked this to organization page because there are two repositories for the app, one for iOS, one for Android... Both are pinned on org page.

Can change the link to point to one of them if you prefer that

Reply:
Yes, please - [here's an example](https://github.com/bitcoin-dot-org/bitcoin.org/blob/master/_wallets/bither.md) of how to link to multiple repositories.

Reply:
> Yes, please - [here's an example](https://github.com/bitcoin-dot-org/bitcoin.org/blob/master/_wallets/bither.md) of how to link to multiple repositories.

thanks, done!